source/ui/acquaman/AMGenericStepScanConfigurationView.cpp

```diff
@@ -132,7 +132,6 @@ AMGenericStepScanConfigurationView::AMGenericStepScanConfigurationView(AMGeneric
 	moreLayout->addStretch();
 	moreLayout->addLayout(layout);
 	moreLayout->addLayout(detectorsAndI0Layout);
-	moreLayout->addWidget(i0ComboBox_);
```

Comment:
Most of the time, the box would show up as normal in the configuration view, but every once in a while I noticed it showed up in kind of a funny place. It looks like it was added to two different layouts, and that was causing the funny behavior. I kept the one that I think makes sense (next to the I0 label), this is the one that was removed.